src/OWASPZAPDotNetAPI/NugetPackaging/OWASPZAPDotNetAPI.nuspec

```diff
@@ -2,17 +2,20 @@
 <package >
   <metadata>
     <id>OWASPZAPDotNetAPI</id>
-    <version>1.0.0</version>
+    <version>2.6.0</version>

```

Comment:
@kingthorin  Yes, I thought it would be easy to reference the API Clients corresponding to the ZAP version.

It is just easy for people to identify and download. Let's say we release a new ZAP version 2.7, then it becomes natural to download the dot net api client 2.7 version.

I don't quite get the point about "a bug in the api implementation itself" and how it would be out of sync. Forgive my ignorance.